src/controllers/user.controller.ts

```diff
@@ -15,11 +15,11 @@ import {
   requestBody,
   response
 } from '@loopback/rest';
-import {Keyring} from '@polkadot/api';
-import {u8aToHex} from '@polkadot/util';
-import {encodeAddress} from '@polkadot/util-crypto';
-import {KeypairType} from '@polkadot/util-crypto/types';
-import {polkadotApi} from '../helpers/polkadotApi';
+// import {Keyring} from '@polkadot/api';
```

Comment:
mendingan all of the polkadot logic di extract ke file terpisah terus di comment 1 line aja